src/rust/engine/options/src/arg_splitter_tests.rs

```diff
@@ -288,3 +305,91 @@ fn test_split_args_short_flags() {
         shlex_and_split_args(None, "pants -x run path/to:bin")
     );
 }
+
+#[test]
+fn test_help() {
+    fn assert_help(args_str: &str, expected_goals: Vec<&str>, expected_specs: Vec<&str>) {
```

Comment:
`#[track_caller]` -- makes the caller show up in any panic as the location

Reply:
Neat!